docs/src/configuration/routes/_index.md

```diff
@@ -66,80 +66,98 @@ so consider using a path like `https://{default}/api` instead.
-* A placeholder URL such as `https://{default}/blog`
+* An absolute URL such as `https://example.com/blog`
+* A URL with a placeholder such as `https://{default}/blog`
 
-Placeholders can be either `{default}` or `{all}`, and they map to the [custom domains](../../domains/quick-start.md) that 
-you have defined in your project settings.
+The available placeholders are `{default}` and `{all}`.
+They stand in for the [custom domains](../../domains/quick-start.md) you've defined in your project.
 
-These domains can be top level domains (`example.com`) or subdomains (`app.example.com`).
+These domains can be top-level domains (`example.com`) or subdomains (`app.example.com`).
 
 ### `{default}`
 
 `{default}` represents your default custom domain.
+If you have set your default domain to `example.com`,
+`example.com` and `{default}` in your `.platform/routes.yaml` file have the same result for your Production environment.
 
-Assuming that you set your default domain to `example.com`.
+You can use the `{default}` placeholder:
 
-Using both `example.com` and `{default}` in your `.platform/routes.yaml` file generates the same URLs for your **production** environment.
+```yaml {location=".platform/routes.yaml"}
+"https://{default}/blog":
+    type: upstream
+    upstream: "app:http"
+```
 
-However, your **development** environment URLs differ depending on the method you use.
+And you can use an absolute URL:
 
-1. If you use the default placeholder:
+```yaml {location=".platform/routes.yaml"}
+"https://example.com/blog":
+    type: upstream
+    upstream: "app:http"
+```
+
+In both cases, the URLs for your Production environment are the same.
+
+#### URLs in non-Production environments
+
+URLs in non-Production environments differ depending on whether or not you use the `{default}` placeholder.
+
+If you use the `{default}` placeholder:
 
 ```yaml {location=".platform/routes.yaml"}
-"https://{default}/":
+"https://{default}/blog":
     type: upstream
     upstream: "app:http"
 ```
 
-The generated development URL of the `feature` environment is:
+The generated URL for the `feature` environment is:
 
 ```txt
 https://feature-t6dnbai-abcdef1234567.us-2.platformsh.site/blog
 ```
 
-2. If you use the explicit domain method:
+If you use an absolute URL:
 
 ```yaml {location=".platform/routes.yaml"}
-"https://example.com/":
+"https://example.com/blog":
     type: upstream
     upstream: "app:http"
 ```
 
-The generated development URL of the `feature` environment is:
+The generated URL for the `feature` environment is:
 
 ```txt
 https://example.com.feature-t6dnbai-abcdef1234567.us-2.platformsh.site/blog
 ```
 
-WARNING: THIS INCONSISTENT BEHAVIOR IS FIXED IN THE UPCOMING RELEASE.
-
-Using both `example.com` and `{default}` in your `.platform/routes.yaml` file generates the same URLs for your **production** and **development** environments.
+{{< note theme="warning" title="This behavior will change in an upcoming release" >}}
```

Comment:
I think it would be better to have "Warning!" in the text, I think it attracts more attention.
And I am not sure, but could we condense the message a bit?

...
At that point, the inconsistency is fixed, both `example.com` and `{default}` generate the same URL, If you have set your default domain to `example.com`. E.g.:

```txt
https://example.com.feature-t6dnbai-abcdef1234567.us-2.platformsh.site/blog
```

If you haven't set a default domain, the `{default}` URL is generated in the old way:

```txt
https://feature-t6dnbai-abcdef1234567.us-2.platformsh.site/blog
```

This leaves out the `example.com` prefix that you find if you use an absolute URL.

Reply:
I think I was editing it with a mind of using the text in the future, so I didn't include text like "the old way" with the idea that it would come out of the note and replace the other text. I can look again at it to simply it, though. Always in favor of that :+1: